projects/mars-theme/package.json

```diff
@@ -20,6 +20,7 @@
     "@frontity/mars-theme": "^1.3.1",
     "@frontity/tiny-router": "^1.1.0",
     "@frontity/wp-source": "^1.5.0",
+    "@frontity/amp": "0.0.1",
```

Comment:
Same as above: was it just for testing or did you want to add amp to mars-theme for some specific reason? :)

Reply:
```suggestion
```
Does this work?